src/Components/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.csproj

```diff
@@ -6,6 +6,8 @@
     <PackageTags>$(ComponentEfCorePackageTags) postgressql postgres npgsql sql</PackageTags>
     <Description>A PostgreSQL® provider for Entity Framework Core that integrates with Aspire, including connection pooling, health checks, logging, and telemetry.</Description>
     <PackageIconFullPath>$(SharedDir)PostgreSQL_logo.3colors.540x557.png</PackageIconFullPath>
+    <!-- We need to ingest the stable version of EF and npgsql in order for us to produce stable  bits. -->
+    <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
```

Comment:
FYI: @eerhardt will need to figure out what to do about this one given the npgsql dependency.